### PR TITLE
NG exporter

### PR DESCRIPTION
**Pull request by ghost**
Added exporter to RalphNG.

connected to #1410

### CODE REVIEW

**Comment by mkurek** [comment]
what's the difference between `export_model` and `export_models`?

**Comment by ghost** [comment]
`export_models` will export all models required by NG, now it's just ordinary copy-paste with small change.

File: src/ralph/export_to_ng/management/commands/export_model.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+class Command(BaseCommand):
+    help = os.linesep.join([
+        'Export data in Ralph-NG format.',
+        ALLOWED_MODELS_MSG,
+    ])
+
+    # TODO:: zipped as arg
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--dst-dir',
```
Comment (mkurek) [inline src/ralph/export_to_ng/management/commands/export_model.py]: afaik if param has only `--` version it is not required - shouldn't it be required?

File: src/ralph/export_to_ng/management/commands/export_model.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+    def handle(self, *args, **options):
+        models = args
+        if not models:
+            models = SIMPLE_MODELS + DEPENDENT_MODELS + MANY_TO_MANY
+        for model in models:
+            self.stdout.write('Processing model {}\n'.format(model))
+
+            model_resource = get_resource(model)
+            queryset = model_resource._meta.model.objects.all()[:1]
```
Comment (mkurek) [inline src/ralph/export_to_ng/management/commands/export_model.py]: is `[:1]` only for tests?

File: src/ralph/export_to_ng/management/commands/export_model.py (hunk 1 above)
Comment (ghost) [inline src/ralph/export_to_ng/management/commands/export_model.py]: In current version should be I think. I'll fix it, thanks.

File: src/ralph/export_to_ng/management/commands/export_model.py (hunk 2 above)
Comment (ghost) [inline src/ralph/export_to_ng/management/commands/export_model.py]: True

**Comment by ar4s** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**